### ISSUE TITLE
[Bug] Watching Videos from Main page when set to "feed page" sometimes crashes?

### ISSUE THREAD

**Issue by SudoLinux** [TeamNewPipe/NewPipe#892]
I sadly don't have much info on this, do I need to submit the newpipe bug reports through the apps auto bug tracker? Anyway, when you click on videos if you set Home page to "Feed Page" it sometimes crashes with "guru meditation." when you click upon a video, may be duplicate, but wasn't sure. thanks!

**Comment by wb9688** [comment]
Yeah, you should report it with NewPipe's bug reporter. I haven't had this
issue myself, but I don't really use the feed myself atm. Btw we can't do
much without a stacktrace (which NewPipe's bug reporter includes).

**Comment by SudoLinux** [comment]
I knew it, I'll try and cause the bug, its not that hard actually

**Comment by SudoLinux** [comment]
@wb9688 can't seem to force it, it just happens once a day for me, is there a folder for bug reports I can access?

**Comment by theScrabi** [comment]
No. You can send us the bugreport via mail, or save it via share menu.

**Comment by SudoLinux** [comment]
OK, I finally managed to get it happen, here is the big boom



{
   "user_action": "ui error",
   "request": "App crash, UI failure",
   "content_language": "en",
   "service": "none",
   "package": "org.schabi.newpipe",
   "version": "0.11.0",
   "os": "Linux Android 6.0.1 - 23",
   "time": "2017-12-07 01:18",
   "ip_range": "184.170.0.0",
   "exceptions": [
      "java.lang.IllegalArgumentException: No view found for id 0x7f1000b5 (org.schabi.newpipe:id\/fragment_holder) for fragment VideoDetailFragment{7e299f2 #2 id=0x7f1000b5}\n\tat android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1413)\n\tat android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1740)\n\tat android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1809)\n\tat android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:799)\n\tat android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2580)\n\tat android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2367)\n\tat android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2322)\n\tat android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2229)\n\tat android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700)\n\tat android.os.Handler.handleCallback(Handler.java:739)\n\tat android.os.Handler.dispatchMessage(Handler.java:95)\n\tat android.os.Looper.loop(Looper.java:148)\n\tat android.app.ActivityThread.main(ActivityThread.java:5458)\n\tat java.lang.reflect.Method.invoke(Native Method)\n\tat com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)\n\tat com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)\n"
   ],
   "user_comment": ""
}

**Comment by coffeemakr** [comment]
Found the sentry entry: https://sentry.schabi.org/sentry/newpipe/issues/710/

**Comment by wb9688** [comment]
It happens if the screen is rotated. It shouldn't be hard to fix.

**Comment by SudoLinux** [comment]
OK I changed title and description, looks like I've been accidentally rotating when selecting a video right when selecting it...

**Comment by theScrabi** [comment]
Please do not just dump the json in here. We have no way to read this.

**Comment by SudoLinux** [comment]
@theScrabi my apologies
